src/App.js

```diff
@@ -16,26 +21,22 @@ function App() {
 	};
 
 	return (
+		<ProductContext.Provider value={{ products, addItem }}>
+		<CartContext.Provider value={cart}>
 		<div className="App">
-			<Navigation cart={cart} />
-
-			{/* Routes */}
-			<Route
-				exact
-				path="/"
-				render={() => (
-					<Products
-						products={products}
-						addItem={addItem}
-					/>
-				)}
-			/>
-
-			<Route
-				path="/cart"
-				render={() => <ShoppingCart cart={cart} />}
-			/>
+		<Navigation />
+		
+		{/* Routes */}
+		<Route exact path="/" component={Products} />
+		)}
+		/>
+		
+		<Route
+		path="/cart" component={cart}
```

Comment:
```suggestion
		path="/cart" component={ShoppingCart}
```
This was causing an error.